__tests__/endpoint.test.js

```diff
@@ -0,0 +1,42 @@
+// Require in supertest
+const request = require("supertest");
+//Require in app.js
+const { app } = require("../app/app.js");
+// Require in connection file:
+const db = require("../db/connection");
+// Require in Seed File:
+const seed = require("../db/seeds/seed");
+
+//Require in test data:
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET: /api/topics", () => {
+  test("Responds with 200 and array of objects containing two properties: slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual(
+          expect.arrayContaining([
```

Comment:
This wouldn't check each object within the array, you would need to iterate through the array then you can check for an object containing.
Remember before doing so, you should check the length of the array thats coming back is what you expect as you may get a false positive.